demos/classification_benchmark_demo/cpp_gapi/include/custom_kernels.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <list>
+#include <map>
+#include <string>
+#include <vector>
+
+#include <cpp/ie_cnn_network.h>
```

Comment:
I don't think you need this header there

Reply:
ok